src/main/java/com/example/java/test/junior/developer/dto/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package com.example.java.test.junior.developer.dto;
+
+import lombok.*;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@AllArgsConstructor
+@Builder
+public class UserRequestDto {
+
+    @NonNull
+    private final Long id;
+
+    @NonNull
+    private final String firstName;
+
+    @NonNull
+    private final String lastName;
+
+    @Email(message = "Email should be valid")   // Використовувати javax.validation для верифікації ?
+    @NotNull
+    private final String email;
+
+    @NonNull
+    private final String password;  // Чи можна зберігати тут тимчасово пароль, чи безпечно це?
```

Comment:
In our case - no)) Since we are using http. In real life - you will have https, first of all. Secondly, there are some additional EXTRA measures that can be done to avoid any manipulations with heap dump - but I am 100% sure we don't need dive deep into it.
For US (junior devs making pet project) - this is OK.